packages/framework/src/Form/CategoriesType.php

```diff
@@ -84,4 +94,24 @@ public function getParent(): ?string
     {
         return CollectionType::class;
     }
+
+    /**
+     * @param array $options
+     * @return string|null
+     */
+    private function getMainCategoryPath(array $options): ?string
+    {
+        if ($options['product'] === null) {
+            return null;
+        }
+
+        try {
+            $domainConfig = $this->domain->getDomainConfigById($options['domain_id']);
+            $categoriesInPath = $this->categoryFacade->getCategoryNamesInPathFromRootToProductMainCategoryOnDomain($options['product'], $domainConfig);
+
+            return implode(' > ', $categoriesInPath);
```

Comment:
it's a little strange that the main category path is translated to the domain language, while categories are in the admin locale. 

![Snímek obrazovky 2025-03-28 v 18 33 52](https://github.com/user-attachments/assets/3eb5822d-75ed-4e51-9544-6e9ecf203b7e)

Reply:
yes! Thanks for pointing this out :pray: I will keep  this open until https://github.com/shopsys/shopsys/pull/3881 is merged and then try to do something about it :relaxed: